project/android/app/src/main/AndroidManifest.xml

```diff
@@ -4,7 +4,7 @@
 
    <application
         android:label="AnimeGo"
-        android:icon="@mipmap/ic_launcher">
```

Comment:
This shouldn't get removed because it is the app icon. See if the app icon is still on your home screen.